spec/shared_examples.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
```

Comment:
I think you don't need to require spec_helper here, also this should be under `spec/support` - RSpec loads support folder in spec_helper.